tests/status.py

```diff
@@ -72,6 +72,35 @@ def runTest(self):
         board.expect(prompt)
         self.result_message = 'ifconfig shows ip addresses: %s' % tmp
 
+class MacAddressCheck(rootfs_boot.RootFSBootTest):
+    '''Checked MAC addresses for all interfaces.'''
+    def runTest(self):
+        board.sendline('\nhexdump -n 6 -v -e \'/1 "%02X"\' /proc/device-tree/uccp@18480000/mac-address0')
+        board.expect('0019F5890AFB.*')
+        board.expect(prompt)
+        board.sendline('\nhexdump -n 6 -v -e \'/1 "%02X"\' /proc/device-tree/uccp@18480000/mac-address1')
+        board.expect('0019F5890AFC.*')
+        board.expect(prompt)
+        board.sendline('\nhcitool dev | grep 00:19:F5:89:0A:FD')
```

Comment:
Because the previous code was checking exact MACs it was technically checking that the correct MAC was being loaded in. The current code doesn't check this so it would pass even if the same MAC was assigned to every interface e.g. I would expect this to pass:
```
WIFI0:       00:19:F5:89:00:01
WIFI1:       00:19:F5:89:00:02
bluetooth:   00:19:F5:89:00:03
ETH0:        00:19:F5:89:00:04
mesh:        00:19:F5:89:00:05
6lowpan:     00:19:F5:89:00:06
```
but this to fail for a number of reasons:
```
WIFI0:       00:19:F5:89:00:01
WIFI1:       00:19:F5:89:00:02
bluetooth:   00:19:F5:89:00:03
ETH0:        00:19:F5:89:00:01
mesh:        00:19:F5:89:00:07
6lowpan:     00:19:F5:89:00:08
```
because we know that all our MACs should be a sequential list generated from the primary mac wifi0.